express-notes-api/server.js

```diff
@@ -0,0 +1,75 @@
+import express from 'express';
+import { getFileContent, updateDatabase } from './fileUtils.js';
+import { validateContent, validateId } from './validationUtils.js';
+
+const app = express();
+const PORT = 8080;
+const NOTES_URL = '/api/notes';
+const data = JSON.parse(await getFileContent('data.json'));
+const { notes } = data;
```

Comment:
While reading the data once at startup saves on request time, it also tends to result in stale data, particularly when multiple clients are making requests or multiple instances of web servers are using a shared data source. Instead, read a fresh copy of the data source with each request.